docs/source/usersguide/input.rst

```diff
@@ -1264,6 +1264,61 @@ The ``<tally>`` element accepts the following sub-elements:
           two post-collision energy bins will be created, one with energies
           between 0 and 1 MeV and the other with energies between 1 and 20 MeV.
 
+        :mu:
+          A monotonically increasing list of bounding **post-collision**
+          cosines of the change in a particle's angle (i.e., :math:`\mu`),
+          which represents a portion of the possible values of :math:`\[-1,1\]`.
```

Comment:
When I try to build the docs, it complains about these escaped brackets and parentheses. When I remove them, it builds fine (both html and latexpdf).